### DISCUSSION TITLE
Renovate shows invalid-config with no any explanation with DEBUG log level

### DISCUSSION THREAD

**Discussion by ezh** [renovatebot/renovate#33200] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

docker container, 39.75.0

### Please tell us more about your question or problem

I have a number of git tags `app-1.2.3.4`, `app-1.2.3.5` ... and version in Dockerfile `1.2.3.0`. Why renovate shows `invalid-config`.

```
           {
             "deps": [
               {
                 "packageName": "https://gitlab/repo.git",
                 "currentValue": "0.1.2.3",
                 "datasource": "git-tag",
                 "versioning": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)\\.(?<build>\\d+)$",
                 "extractVersion": "^app-(?<version>\\d+\\.\\d+\\.\\d+\\.\\d+)$",
                 "replaceString": "ENV APP_VERSION=0.1.2.3",
                 "depName": "https://gitlab/repo.git",
                 "updates": [],
                 "warnings": [],
                 "skipReason": "invalid-config"
               }
             ],
             "matchStrings": [
               "ENV APP_VERSION=(?<currentValue>\\d+\\.\\d+\\.\\d+\\.\\d+)"
             ],
             "packageNameTemplate": "https://gitlab/repo.git",
             "datasourceTemplate": "git-tag",
             "versioningTemplate": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)\\.(?<build>\\d+)$",
             "extractVersionTemplate": "^app-(?<version>\\d+\\.\\d+\\.\\d+\\.\\d+)$",
             "packageFile": "docker/app/Dockerfile"
           }
```

I aim to drop the `app-` prefix from the tag and match it with the actual version in the Dockerfile.

**Comment by ezh** [comment]
There are no relevant logs. I checked all the lines with DEBUG log level.

**Reply by rarkins** [reply]
Could be your regex versioning. Check the source code for that string

**Reply by ezh** [reply]
@rarkins, I need help to understand your proposal.

I checked `versioning` with regex101 https://regex101.com/r/ewyGyJ/1
It works.

I see `"currentValue": "0.1.2.3",` so currentValue works well.

https://github.com/renovatebot/renovate/blob/main/lib/workers/repository/process/lookup/index.ts#L92
      res.skipReason = 'invalid-config';

Is there something wrong with `isGetPkgReleasesConfig`
https://github.com/renovatebot/renovate/blob/main/lib/modules/datasource/common.ts#L43

**Reply by rarkins** [reply, marked as answer]
"git-tag" is an invalid datasource

**Reply by ezh** [reply]
@rarkins Thank you very much, man! You made my day. I went in the wrong direction and lost there.
